src/Presence.js

```diff
@@ -73,6 +73,11 @@ class Presence {
         }
         var old_state = this.state;
         this.state = newState;
+
+        if (MatrixClientPeg.get().isGuest()) {
+            return; // don't try to set presence when a guest; it won't work.
+        }
+
```

Comment:
As I understand this will soon be abled - is there any harm in sending it anyway in the meantime?